lib/topic_assigner.rb

```diff
@@ -110,6 +112,9 @@ def assign(assign_to, silent: false)
       user_ids: staff_ids
     )
 
+    message = AssignMailer.send_assignment(assign_to.email, @topic, @assigned_by)
```

Comment:
Should this unconditionally send out the email even if the user mutes the topic?

Reply:
Let me answer, @spog163 and I worked on this improvement. Currently it sends emails always without an option to disable. Our task was pointed to staff users, who need to have an email notification always. Agree, that this setting must be an option (disabled by default) and depend on mute conditions. So, let us to make some updates first :)